### ISSUE TITLE
Update docker-compose.yml version

### ISSUE THREAD

**Issue by anastis** [cytopia/devilbox#618]
#### ISSUE TYPE

 - Feature request

#### SUMMARY

Update the docker-compose.yml file to a newer version, to support newer features. 

#### Goal

I would like to be able to define bind mounts using the long syntax, so that I can for example configure the propagation parameter, so that a mounted folder can appear inside the default data/www mount.
The specific feature (long format mounts) requires docker-compose version 2.3, but I guess bringing it as much as up to date possible wouldn't hurt.

**Comment by anastis** [comment]
To clarify, I'm actually using a docker-compose.override.yml so that my mounts will survive an update, however it must follow the docker-compose.yml's version, therefore I can't use the long syntax.

**Comment by Tuurlijk** [comment]
Hi @anastis can you make a pull request?

**Comment by anastis** [comment]
I think I shouldn't... I've only used Docker due to devilbox, and it's only been a few days, so I really don't know what I'm doing :smile: 

Meanwhile I've worked around the issue I was facing by copying my files deep inside the data/www path where I needed them. Symlinks outside of data/www wouldn't work. It looks like the propagation parameter is the only thing that can provide a symlink experience between the host and the container.
